src/components/map/HfpMarkerLayer.js

```diff
@@ -31,12 +32,18 @@ class HfpMarkerLayer extends Component {
 
     for (; posIdx < total; posIdx++) {
       const position = positions[posIdx];
-      let prevDifference = 30;
+      // This acts as the upper limit for when a time matches a marker.
+      // If it is too low, markers for selected journeys might not match.
+      let prevDifference = 180;
```

Comment:
Even 3 minutes is not enough to show markers in all cases. I wouldn't go higher than this.

Reply:
agree